Author: Adam Buchbinder <adam.buchbinder@gmail.com>
Description:
 Add support for NUT Container format, using the format's magic string.
 (Closes: #528242).

diff -Naurp file.orig/magic/Magdir/animation file/magic/Magdir/animation
--- file.orig/magic/Magdir/animation	2009-05-11 11:39:41.000000000 +0000
+++ file/magic/Magdir/animation	2009-05-11 12:41:14.000000000 +0000
@@ -813,3 +813,8 @@
 >>51	byte&0x20	!0	stereo
 #>>51	byte&0x10	0	FFT
 #>>51	byte&0x10	!0	DCT
+
+# Type:	NUT Container
+# URL:	http://wiki.multimedia.cx/index.php?title=NUT
+# From:	Adam Buchbinder <adam.buchbinder@gmail.com>
+0	string	nut/multimedia\ container\0	NUT multimedia container
